drivers/stepper/gpio_stepper/Kconfig

```diff
@@ -0,0 +1,11 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 Carl Zeiss Meditec AG
+# SPDX-License-Identifier: Apache-2.0
+
+menu "GPIO STEPPER Driver"
+
+config GPIO_STEPPER
+	bool "Activate driver for GPIO_STEPPER_MOTOR_CONTROLLER"
+	help
```

Comment:
this is missing DT automatic enablement

Reply:
you mean depends on DT_HAS_STEPPER_ENABLED?

Reply:
`DT_HAS_GPIO_STEPPER_ENABLED`, also, a `build_all` like test to make sure it compiles just fine (see eg input subsys, sensors, regulators, etc.)

Reply:
Hi @gmarull, i have added the build_all test. Could you have a brief look at it as well